cilium/CFP-32810-hybrid-routing-mode.md

```diff
@@ -0,0 +1,83 @@
+# CFP-32810: Add hybrid routing mode in Cilium
+
+**SIG: SIG-NAME** SIG-Clustermesh
+
+**Begin Design Discussion:** 2025-06-16
+
+**Cilium Release:** N/A
```

Comment:
is there a release you want to target?

Reply:
Preferably 1.18, but I guess it would depend on feedback on this CFP and what the final scope of the work would be.